libs/core/src/components/pds-checkbox/pds-checkbox.scss

```diff
@@ -1,21 +1,21 @@
 :host {
-  --background-color: var(--pine-color-base-white);
-  --background-color-disabled: var(--pine-color-neutral-grey-200);
-  --background-color-hover: var(--pine-color-neutral-grey-100);
-  --border: var(--pine-border-interactive);
-  --border-color-disabled: var(--pine-color-neutral-charcoal-100);
-  --border-color-hover: var(--pine-border-interactive-hover-color);
-  --border-color-icon: var(--pine-color-base-white);
-  --border-radius: var(--pine-border-radius-xs);
+  --border: var(--pine-border-width-thin) solid var(--pine-color-grey-400);
```

Comment:
Could we add `default` here at the end? What do you think?

Reply:
Works for me!